apps/backoffice-v2/src/common/components/molecules/DateRangePicker/DateRangePicker.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, ComponentProps, useEffect } from 'react';
+import { CalendarIcon } from '@radix-ui/react-icons';
+import { formatDate } from '@/common/utils/format-date';
+import { ctw } from '@/common/utils/ctw/ctw';
+import { Button } from '../../atoms/Button/Button';
+import { Calendar } from '../../organisms/Calendar/Calendar';
+import { Popover, PopoverContent, PopoverTrigger } from '@ballerine/ui';
+import { DateRange } from 'react-day-picker';
+
+type TDateRangePickerProps = {
+  onChange: ComponentProps<typeof Calendar>['onSelect'];
+  value: ComponentProps<typeof Calendar>['value'];
+  className: ComponentProps<'div'>['className'];
+};
+
+export const DateRangePicker = ({ onChange, value, className }: TDateRangePickerProps) => {
+  const [date, setDate] = useState<DateRange | undefined>();
+  console.log(value);
```

Comment:
```suggestion
```

Reply:
I just used that for debugging, to check if the value is being updated or not. I should have been excluded, but I have made a stupid mistake. I will keep that in mind next time. 